tests/unit/admin/views/test_banners.py

```diff
@@ -0,0 +1,25 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from warehouse.admin.views import banners as views
+from warehouse.banners.models import Banner
+
+from ....common.db.banners import BannerFactory
+
+
+class TestBannerList:
+    def test_list_all_banners(self, db_request):
+        [BannerFactory.create() for _ in range(5)]
```

Comment:
Sorry, I missed this because it was marked as outdated by github! I wasn't aware of this `create_batch` method and I'll definitely update the tests to use it :+1:

Reply:
Changed in c988b898f464df5e538ff3d32af46b191f18988b